repos/system_upgrade/el7toel8/actors/detectgrubconfigerror/actor.py

```diff
@@ -37,4 +37,6 @@ def process(self):
                 reporting.RelatedResource('file', config)
             ])
 
-        self.produce(GrubConfigError(error_detected=error_detected))
+        config_error = GrubConfigError(error_detected=error_detected,
+                                       error_description='GRUB_CMDLINE_LINUX syntax')
+        self.produce(config_error)
```

Comment:
I would advocate to produce this conditionally as well. Please, check that whoever consumes grub errors checks whether  the error_detected is True. If so, the message can be produced conditionally.